opt-tools/docker/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '3'
+
+services:
+
+  postgres_db:
+    image: postgres
+    container_name: posgres_db
+    restart: always
+    environment:
+      - POSTGRES_USER=postgres    
+      - POSTGRES_PASSWORD=school
+      - POSTGRES_DB=school
+    volumes:
+      - /home/vadim/Docker/data/postgres
+      - /home/vadim/Docker/data/dump:/db-dump
+    ports:
+      - "5432:5432"
+    networks: 
+      - docker-practice-net 
+
+      
+  redis:
+    image: "redis:alpine"
```

Comment:
To be honest, i copy-pasted redis image from here: https://kb.objectrocket.com/redis/run-redis-with-docker-compose-1055

Reply:
Got it. Let's try to remove those `redis-server` overwrite and `REDIS_REPLICATION_MODE` because it adds additional behavior, in particular - `REDIS_REPLICATION_MODE`. I've found this option in Bitnami based images - i'm not sure that we should use this. Probably this env is set with the same value by default - but the preferable way is to do not overwrite things if we do not know why we use them.